docs/devguide/building/dependencies.rst

```diff
@@ -47,6 +47,9 @@ It is included in the release archive of ACE/TAO, which is downloaded by the con
 ACE/TAO
 =======
 
+OpenDDS is compatible with the current DOC Group micro release in the ACE 6.x / TAO 2.x series.
+See the :ghfile:`README.md` file for details.
+
```

Comment:
```suggestion
```

I'm not sure what details about ACE/TAO version are in the README that are not here. This sections is already very detailed.